setup.py

```diff
@@ -37,4 +35,7 @@
     install_requires=(
         'functools32',
     ) if version_info.major == 2 and version_info == 7 else (),
```

Comment:
Will this work ok with the conda recipe if I build a py3 package with miniconda2?